src/main/java/sirius/db/mixing/properties/MultiLanguageStringProperty.java

```diff
@@ -206,6 +207,14 @@ protected boolean isConsideredNull(Object propertyValue) {
         return values.entrySet().stream().anyMatch(entry -> Strings.isEmpty(entry.getValue()));
     }
 
+    @Override
+    public void parseValues(Object entity, Values values) {
+        MultiLanguageString multiLanguageString = getMultiLanguageString(entity);
+        Map<String, String> mlsMap = new HashMap<>(multiLanguageString.data());
+        mlsMap.put("fallback", values.at(0).getString());
```

Comment:
what about MLS fields with fallback disabled?

Reply:
Worse case we have to look at this after the patch. The highest priority now is to bring a working version in OX for the Content entities and they all have fallback enabled.

Reply:
Treating MLS fields without fallback is fixed in https://github.com/scireum/sirius-db/pull/405: https://github.com/scireum/sirius-db/pull/405/commits/ff2bacd66ec9cb0ad1aa713543558e45043f0edf